pages/quiz/index.tsx

```diff
@@ -5,21 +5,59 @@ import React, { useContext } from 'react'
 import styles from '../../styles/Home.module.css'
 import { AES } from 'crypto-js'
 import { AppContext } from '../../context/context'
+import { Accessibility, Citizenship, EmploymentStatus, Family, Insurance, SurveyAnswers } from '../../models/types'
+import * as Yup from 'yup'
+import { Formik, Form, Field, ErrorMessage, FormikHelpers } from "formik";
+
+const errorMessages = {
+  incomeError: 'Please enter a positive number for income',
+  ageError: 'Please enter a valid age'
+}
+
 
 const Quiz: NextPage = () => {
 
+
   const quizState = useContext(AppContext)
 
   const router = useRouter();
 
-  const handleSubmit = async (e: React.SyntheticEvent) => {
-    e.preventDefault();
+  const validationSchema = Yup.object({
+    income: Yup.number()
+      .positive(errorMessages.incomeError)
+      .typeError(errorMessages.incomeError),
+    language: Yup.string(),
+    citizenship: Yup.string(),
+    parentAge: Yup.number()
+      .positive(errorMessages.ageError)
+      .integer(errorMessages.ageError)
+      .typeError(errorMessages.ageError),
+    childAge: Yup.number()
+      .positive(errorMessages.ageError)
+      .integer(errorMessages.ageError)
+      .typeError(errorMessages.ageError),
+    family: Yup.string(),
+    employmentStatus: Yup.number().min(1).max(10),
```

Comment:
I managed to merge the branches and they work together now, however I had to remove the `.required()` part from employmentStatus. Did you guys make this field required for any particular reason?

Reply:
Yup that shouldn't have been there